packages/medusa/src/repositories/product-category.ts

```diff
@@ -44,22 +44,22 @@ export const ProductCategoryRepository = dataSource
       const options_ = { ...options }
       options_.where = options_.where as FindOptionsWhere<ProductCategory>
 
-      const legacySelect = buildLegacyFieldsListFrom(options_.select)
-      const legacyRelations = buildLegacyFieldsListFrom(options_.relations)
+      const columnsSelected = buildLegacyFieldsListFrom(options_.select)
+      const relationsSelected = buildLegacyFieldsListFrom(options_.relations)
```

Comment:
question: How do we get rid of `buildLegacyFieldsListFrom`?

Reply:
Ping :)

Reply:
@olivermrbl the select takes a string | string[], so it is a useful utils for that. I agree that we could rename it to something like `objectToStringPath` and `pathToDeepObject` as suggested by @riqwan

Reply:
Yes, I think that makes sense :) Current naming suggests that it should be removed

Reply:
Created [CORE-1713](https://linear.app/medusajs/issue/CORE-1317/rename-buildlegacyfieldslistfrom-to-objecttostringpath) since this changes quite a bit of files.

Reply:
Makes sense - good call